source/MRMesh/MRMeshBuilder.cpp

```diff
@@ -478,13 +478,15 @@ struct PathOverIncidentVert {
     // all iterators in [vertexBegIt, vertexEndIt) must have the same central vertex
     std::vector<IncidentVert>::iterator vertexBegIt, vertexEndIt;
     size_t lastUnvisitedIndex = 0; // pivot index. [vertexBegIt, vertexBegIt + lastUnvisitedIndex) - unvisited vertices
+    VertBitSet visitedVertices;
```

Comment:
I think this BitSet better create only once for the whole deduplicate algorithm

Reply:
done